certified-connectors/TranslatorV3/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1870 @@
+            "in": "query",
+            "description": "The name of the fromScript within the specified language.",
+            "required": false,
+            "x-ms-summary": "From script",
+            "x-ms-visibility": "advanced",
+            "x-ms-dynamic-values": {
+              "operationId": "GetTransliterationFromScripts",
+              "parameters": {
+                "language": {
+                  "parameter": "from"
+                }
+              },
+              "value-path": "Code",
+              "value-title": "Name"
+            },
+            "x-ms-test-value": "Latn",
+            "type": "string"
+          },
+          {
+            "name": "toScript",
+            "in": "query",
+            "description": "The name of the toScript within the specified language.",
+            "required": false,
+            "x-ms-summary": "To script",
+            "x-ms-visibility": "advanced",
+            "x-ms-dynamic-values": {
+              "operationId": "GetTransliterationToScriptsForLanguage",
+              "parameters": {
+                "language": {
+                  "parameter": "to"
+                }
+              },
+              "value-path": "Code",
+              "value-title": "Name"
+            },
+            "x-ms-test-value": "Latn",
+            "type": "string"
+          },
+          {
+            "name": "category",
+            "in": "query",
+            "description": "The translation category (default: 'general').",
+            "required": false,
+            "x-ms-summary": "Category",
+            "x-ms-visibility": "advanced",
+            "default": "general",
+            "type": "string"
+          },
+          {
+            "name": "allowFallback",
+            "in": "query",
+            "description": "Specifies that the service is allowed to fall back to a general system when a custom system doesn't exist.",
+            "required": false,
+            "x-ms-summary": "Allow fallback",
+            "x-ms-visibility": "advanced",
+            "default": "false",
+            "type": "string",
+            "enum": [
+              "true",
+              "false"
+            ]
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "Text": {
+                    "type": "string",
+                    "title": "Text",
+                    "description": "Text to translate.",
+                    "x-ms-visibility": "important",
+                    "default": "Hello"
+                  }
+                },
+                "required": [
+                  "Text"
+                ]
+              },
+              "required": [
+                "items"
+              ]
+            }
+          }
+        ],
+        "externalDocs": {
```

Comment:
@HamshavathiM the "externalDocs" is not required, can you please remove it from all the swagger file?